doc/common/chapter00.rst

```diff
@@ -130,15 +130,27 @@ Architectural Principles
 
 Following are a number of key architectural principles that apply to all Reference Architectures produced by the Anuket project:
 
-1. **Open source preference:** To ensure, by building on technology available in open source projects, that suppliers’ and operators’ investment have a tangible pathway towards a standard and production ready Cloud Infrastructure solution portfolio.
-2. **Open APIs:** To enable interoperability and component substitution, and minimize integration efforts by using openly published API definitions.
-3. **Separation of concerns:** To promote lifecycle independence of different architectural layers and modules (e.g. disaggregation of software from hardware).
-4. **Automated lifecycle management:** To minimize costs of the end-to-end lifecycle, maintenance downtime (target zero downtime), avoid errors and discrepancies resulting from manual processes.
-5. **Automated scalability:** To minimize costs and operational impacts through automated policy-driven scaling of workloads by enabling automated horizontal scalability of workloads.
-6. **Automated closed loop assurance:** To minimize operational costs and simplify Cloud Infrastructure platform operations by using automated fault resolution and performance optimization.
-7. **Cloud nativeness:** To optimise the utilization of resources and enable operational efficiencies.
-8. **Security compliance:** To ensure the architecture follows the industry best security practices and is at all levels compliant to relevant security regulations.
-9. **Resilience and Availability:** To allow High Availability and Resilience for hosted VNFs, and to avoid Single Point of Failure.
+1. **Open-source preference:** for building Cloud Infrastructure
+   solutions, components and tools, using open-source technology.
+2. **Open APIs:** to enable interoperability, component
+   substitution, and minimise integration efforts.
+3. **Separation of concerns:** to promote lifecycle independence of
+   different architectural layers and modules (e.g., disaggregation of
+   software from hardware).
+4. **Automated lifecycle management:** to minimise the
```

Comment:
LCM of what? If workloads, then "target zero downtime" is reasonable but if infrastructure then that is not.

Reply:
@pgoyal01 This is about the RM, therefore about infrastructure. Maybe we should remove the zero downtime requirement?